src/pages/outlets/Index.hu.md

```diff
@@ -6,18 +6,26 @@ logo: index.svg
 isIndependent: true
 properties:
   - key: Kategória
-    value: online híroldal
+    value: közéleti híroldal
+  - key: Látogatottság
+    value: 599 627 látogató/nap (2018 július)
+  - key: Megjelenési forma
+    value: online
   - key: Világnézet
-    value: értéksemlegesség
+    value: értéksemleges
+  - key: Státusz
+    value: független
   - key: Főszerkesztő
     value: Tóth-Szenesi Attila
+  - key: Kiadó
+    value: Index.hu Zrt.
   - key: Tulajdonosok
     value: Magyar Fejlődésért Alapítvány (dr. Bodolai László – kurátor)
 contact:
-  - key: Kiadó elérhetőségei
-    value:
+  - name: Kiadó elérhetőségei
```

Comment:
This is still a list (array). Whereas in the typedefinition (and in the logic) it is a simple object.

Reply:
Simply remove the - sign from the beginning:
```
contact:
  name: blah
  fields:
    - key: blah
       value: blah
```